File: riak/ringo/delete/riak_delete.js
#! /usr/bin/ringo
// ------------------------------------------------------------------
//
//	riak_delete.js
//
//						Feb/24/2015
//
// ------------------------------------------------------------------
var text_manipulate = require ('/var/www/data_base/common/ringo_common/text_manipulate.js');
// ------------------------------------------------------------------
print	("*** 開始 ***");
var args = require('system').args;
var key_in = args[1];
print	(key_in);

var client= require ("ringo/httpclient");

var url_target='http://host_ubuntu1:8098/riak/shimane/' + key_in;

var data=client.del (url_target);

print	("*** 終了 ***");
// ------------------------------------------------------------------

File: riak/jython/delete/riak_delete.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	delete/riak_delete.py
#
#					Jun/25/2015
# ----------------------------------------------------------------
import	os
import sys
reload(sys)
sys.setdefaultencoding('utf-8')
#
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_get_uri import rest_delete_proc
#
# ----------------------------------------------------------------
print ("*** 開始 ***")
#
key_in = sys.argv[1]
print ("%s" % key_in)
#
url_target = 'http://host_ubuntu1:8098/riak/shimane/' + key_in
#
rest_delete_proc (url_target)
#
print ("*** 終了 ***")
# ----------------------------------------------------------------
